account/identity.go

```diff
@@ -12,20 +13,62 @@ import (
 	"golang.org/x/net/context"
 )
 
-// Identity ddenDescribes a unique Person with the ALM
+const (
+	// KeycloakIDP is the name of Keycloak Identity Provider
+	KeycloakIDP string = "kc"
+	// GitHubIDP is the name of GitHub Identity Provider
+	GitHubIDP string = "gh"
+)
+
+// NullUUID can be used with the standard sql package to represent a
+// UUID value that can be NULL in the database
+type NullUUID struct {
+	UUID  uuid.UUID
+	Valid bool
+}
+
+// Scan implements the sql.Scanner interface.
+func (u *NullUUID) Scan(src interface{}) error {
+	if src == nil {
+		u.UUID, u.Valid = uuid.Nil, false
+		return nil
+	}
+
+	// Delegate to UUID Scan function
+	u.Valid = true
+
+	switch src := src.(type) {
+	case uuid.UUID:
+		return u.UUID.Scan(src.Bytes())
+	}
```

Comment:
It will be passed to u.UUID.Scan(src). If it's []byte or string it will pass the check. If no then it will fail with an error.